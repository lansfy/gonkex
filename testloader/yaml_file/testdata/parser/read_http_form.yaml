- name: loader MUST read form requests
  method: POST
  headers:
    Content-Type: multipart/form-data
  path: /some/path
  form:
    fields:
      field_name1: value1
      field_name2: value2
  response:
    200: "aaabbb"
