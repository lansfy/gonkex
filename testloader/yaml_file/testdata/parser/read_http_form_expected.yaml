- GetName: loader MUST read form requests
  GetMethod: POST
  Path: /some/path
  ContentType: multipart/form-data
  Headers:
    Content-Type: multipart/form-data
  GetForm:
    GetFields:
      field_name1: value1
      field_name2: value2
  GetResponses:
    200: "aaabbb"
  GetFileName: testdata/parser/read_http_form.yaml
  GetLineNumber: 1
  FirstTestInFile: true
  LastTestInFile: true
